Update/omot_015.txt

```diff
@@ -234,13 +234,13 @@ void main()
 //rせめて連中には私を罵倒した報いを受けさせてやろうと、直接総支配人である葛西に強盗計画のことをタレ込んでやった。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "せめて連中には私を罵倒した報いを受けさせてやろうと、直接総支配人である葛西に強盗計画のことをタレ込んでやった。",
-			NULL, "Someone must've snitched the robbery plan to the general manager, Kasai. Probably hoping to get a reward by throwing me under the bus.", Line_Normal);
+			NULL, "I wanted at least to pay them back for throwing me under the bus, so I snitched the robbery plan to the general manager, Kasai.", Line_Normal);
 	ClearMessage();
 
 //rもちろん、自分のことは伏せておいたのは言うまでもない。…しかし、葛西は薄く笑った後、やおら取り出した拳銃を向けながらこう言い放ったのだ。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "もちろん、自分のことは伏せておいたのは言うまでもない。…しかし、葛西は薄く笑った後、やおら取り出した拳銃を向けながらこう言い放ったのだ。",
-			NULL, "Of course, it goes without saying that I did my part to keep it secret. ...But Kasai lightly laughed at us and then spoke while pulling out a pistol.", Line_Normal);
```

Comment:
Rephrase No.2
”Of course, it goes without saying that I kept my participation secret from him.”